src/Integration/Helpers/Timer/ITimer.cs

```diff
@@ -22,7 +22,7 @@
 
 namespace SonarLint.VisualStudio.Integration
 {
-    public interface ITimer
+    public interface ITimer : IDisposable
```

Comment:
Miscellaneous: I moved the _IDisposable_ from the wrapper class to the interface; we're know we're wrapping a disposable system class, and this way the consuming class doesn't have to do a safe cast to _IDisposable_ before clearing up the timer.

Reply:
I dislike the pattern we have is sonarlint, where IDisposable is not part of the interface. I mean, in practice for a lot of classes we have 1 maybe 2 implementations (+1 in test), of which we know they are IDisposable - so why not mark it on the interface, just like you did here?